tox.ini

```diff
@@ -29,6 +30,15 @@ deps =
     subprocess: subprocess32
 whitelist_externals = open
 
+[testenv:integration-tests]
+deps =
+    tox
+commands =
+    # meta
+    tox "-k test_integration -vvsx" -e pypy-requests
+    tox "-k test_integration -vvsx" -e py27-requests
+    tox "-k test_integration -vvsx" -e py36-requests
```

Comment:
would drop the `-x` here. also clearer form here might be passthru arg style, e.g.

```
tox -e py27-requests -- -k test_integration -vvs
```

would also be good to preserve the ability to handle passthru args when executing this environment, so maybe a further improvement here could be:

```
tox -e py27-requests -- -k test_integration {posargs:-vvs}
...
```

it might also be good to explicitly specify `-k tests/test_integration.py` to avoid inadvertently selecting any tests that contain the substring `test_integration`.